tests/test_token_handling.py

```diff
@@ -317,8 +326,12 @@ def testing_method():
        side_effect=mocked_fetch_public_keys_4, create=True)
 @patch("fabric8a_auth.auth.get_token_from_auth_header",
        side_effect=mocked_get_token_from_auth_header, create=True)
+@patch.dict(os.environ, {"THREESCALE_ACCOUNT_SECRET": "secret1"})
```

Comment:
Could you please create new test scenarios for your feature?
The update to tests has disabled they original purpose.

Reply:
@humaton please review again. Added new tests. 

FYI - the mock function `_mocked_ret_threescale_account_secret_header_none` is required and the environment variable `THREESCALE_ACCOUNT_SECRET` needs to be set for successfully test the authorization based on osio token.